srcpkgs/jrnl/template

```diff
@@ -1,21 +1,17 @@
 # Template file for 'jrnl'
 pkgname=jrnl
-version=1.9.8
-revision=4
+version=2.4.2
+revision=1
 archs=noarch
 build_style=python3-module
-pycompile_module="jrnl"
 hostmakedepends="python3-setuptools"
-depends="python3-cryptography python3-dateutil python3-keyring
-python3-keyrings-alt python3-parsedatetime python3-passlib python3-pytz
-python3-setuptools python3-six python3-tzlocal python3-xdg python3-yaml"
+depends="python3-ansiwrap python3-asteval python3-colorama python3-cryptography
+python3-dateutil python3-keyring python3-parsedatetime python3-passlib
+python3-pytz python3-tzlocal python3-xdg python3-yaml"
```

Comment:
This adds too much noise.

Reply:
to `git-blame` ?

I still end up changing 2/3 lines instead of 3/3